javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+
+var counter = 4;
+
+var timer = setInterval(function () {
+
+  document.querySelector('.countdown-display').textContent = counter;
+  counter -= 1;
```

Comment:
When the first tick of interval completes, the display should be changed from `4` to `3`. Right now, it's re-assigning it `4`.